### ISSUE TITLE
Story 2: Register and Refactor

### ISSUE THREAD

**Issue by allan-stewart** [allan-stewart/react-hogwarts-tdd-kata#26]
@zhon I have updated the README and the `start` repo for the 2nd story. Please take a look and let me know if you find any issues.

I haven't pulled any of the code changes from the `start` repo back to this repository; that seemed a bit unnecessary at this point. But I do think we may want to split a `schedule-page.js` file off from `schedule.js` to be consistent. Also, I'm wondering if there should be an "alert-store" to handle the alerting instead of using the wizard store?

**Comment by zhon** [comment]
I love it! The narrative is excellent. I fixed a couple of typos. 

Boy Scout -> Wizard Scout.

**Comment by zhon** [comment]
> we may want to split a schedule-page.js file off from schedule.js to be consistent

Your right.

Furthermore, the schedule could easily be written with `Catalog` (requires removing registration link) to remove duplication. I am not sure we have time to fit that refactor in the narrative.

**Comment by zhon** [comment]
I feel this story is complete. Unless you can think of anything else.

I will let you close this one.

**Comment by allan-stewart** [comment]
I'll take one last look at this now; I want to go through the whole exercise again now that you've applied the suggestions from the group today. Then I'll close this issue.

**Comment by zhon** [comment]
I hope I didn't mess you up with my quick fixes.

**Comment by allan-stewart** [comment]
No, not really. I found a bunch of places where following the narration was causing some errors in the console -- a few React ones, but mostly linting ones. For example, the narration was still using `var` instead of `let` or `const`. So I fixed those in the `src/` path (but I didn't change any under `test/`).

With these minor narration changes, I was able to walk through the entire exercise via copy/pasting and not get any more errors. So I think this can be called done.

Yay, celebrate! :fireworks:
